### PR TITLE
Extremes clean up

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

File: R/extremes-data.R [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+          ))
+}
+
 #' Create a Flat Text "extremes table" Type Output Table
 #' 
-#' @param rawData An extremes report JSON string.
+#' @param reportObject An extremes report JSON string.
 #' @importFrom dplyr mutate
 #' @return string table
 #' @export
```
Comment (zmoore-usgs) [inline R/extremes-data.R:15]: Does this still need to be exported?

**Review by zmoore-usgs — APPROVED** [review]
Changes look good to me, just the one small comment about whether or not extremesTable needs to be exported.

File: R/extremes-data.R (hunk 1 above)
Comment (thongsav-usgs) [reply R/extremes-data.R:15]: Nope, Rmd/rmarkdown runs inside of the scope of the caller (inside repgen)

Comment (zmoore-usgs) [reply R/extremes-data.R:15]: Gotcha, so should the `@export` tag be removed then?

Comment (thongsav-usgs) [reply R/extremes-data.R:15]: Bah, sorry, I thought you were asking if it should be included. Will take out on next pr.

**Pull request merged**